commands/index.ts

```diff
@@ -94,13 +97,13 @@ export class Commands {
 
   public async autocomplete(message: string): Promise<AutoCompleteResult> {
     // If the line is empty, we show all possible commands as results.
-    if (message == null || message == '') {
+    if (!message) {
       return [this.allCommands(), message]
     }
 
     const [command, query]: (string | undefined)[] = message.trim().split(/\s+/).slice(0)
-    const cmd = await this.find(command)
-    if (cmd) {
+    const cmd = this.find(command)
```

Comment:
some autocompletion is async?

Reply:
yeah, but `this.find` is not

https://github.com/hoprnet/hopr-chat/blob/7c64d9ca1e31b3182270cc2f377e329abf1546b3/commands/index.ts#L76

Reply:
ah, yeah, my bad, misread this,